exercises/sum-of-multiples/src/main/java/SumOfMultiples.java

```diff
@@ -1,3 +1,11 @@
-public class SumOfMultiples {
-    
+class SumOfMultiples {
+
+    SumOfMultiples(int number, int[] set) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getSum() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
```

Comment:
Tricky one. See https://github.com/exercism/xjava/issues/571 for some prior discussion. I tend to think of each exercise's tests + implementations as the entire system, and write code with that in mind. I'm not sure @exercism/java has reached a consensus on this yet, however.